addons/mail/static/src/core/common/message_model.js

```diff
@@ -46,6 +50,24 @@ export class Message extends Record {
     attachment_ids = Record.many("ir.attachment", { inverse: "message" });
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
+    bodyHtml = Record.attr("", {
+        compute() {
+            if (!this.isBodyEmpty) {
+                const htmlDoc = parser.parseFromString(this.body, "text/html");
+                const markdownElements = htmlDoc.querySelectorAll("odoo-markdown");
+                if (!this.store.loader.marked.loaded) {
+                    this.store.loader.marked.load();
+                } else {
```

Comment:
So if the marked library is not loaded, the message is not parsed? 
Shouldn't we await the loading of the lib before rendering markdown?

Reply:
This happens only on loading messages for the 1st time, so this is a minor aspect.
Waiting rendering of messages when markdown is not loaded requires more LOCs or using problematic code like `onWillStart`, so that's why I didn't bother with it.